Source/Mosa.Tool.Launcher/Mosa.Tool.Launcher.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<OutputType>Exe</OutputType>
+		<OutputType>WinExe</OutputType>
```

Comment:
Is this the right output type? Doesn't this make the the launcher Windows only?

Reply:
If I remember right, it only produces a native executable for the current platform.